docs/components/modals/index.html

```diff
@@ -2,11 +2,11 @@
 title: Modals
 minver: 0.2.1
 also:
+  components/box/#scrolling-containers: Box Component ("Scrolling Containers")
```

Comment:
Update link text to `Scrolling Containers (Box)`

Reply:
This will be more consistent to how we linked "Significant White Space (React Compatibility)" in Badges.